### ISSUE TITLE
Disabling default features (audio) causes compile errors for wasm32-unknown-unknown

### ISSUE THREAD

**Issue by toolness** [not-fl3/macroquad#492]
Hello! I was experimenting with disabling the default `audio` feature (via `default-features = false` in my `Cargo.toml`) and it worked fine for my desktop build, but when I ran `cargo build --target wasm32-unknown-unknown` I got this compile error:

```
error[E0599]: no method named `is_loaded` found for struct `dummy_audio::Sound` in the current scope                       --> C:\Users\varma\.cargo\registry\src\github.com-1ecc6299db9ec823\macroquad-0.3.24\src\audio.rs:103:17                  |                                                                                                                   30  |     pub struct Sound {}                                                                                               |     ---------------- method `is_loaded` not found for this                                                        ...                                                                                                                     103 |     while sound.is_loaded() == false {                                                                                |                 ^^^^^^^^^ method not found in `dummy_audio::Sound
```

... that didn't seem to paste very well so here's a screenshot:

![image](https://user-images.githubusercontent.com/124687/189553595-4972d7c9-7dc2-468a-a4c7-c56dcf778201.png)

**Comment by FaunKr** [comment]
I created a pull request with an fix for this #513

**Comment by GRASBOCK** [comment]
I can verify: after the fix by @FaunKr  running the sound example with `macroquad = {default-features = false, version = "0.3"}` in the `Cargo.toml` as expected doesn't give any sound. No warnings no crashes. I think this can be closed then.
